contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
> Why use a different dependency? Is jemalloc different from the other deps at compile time?

Well, it is an optional dependency, so the explanation is a bit more complicated:
 * If USE_BUNDLED_JEMALLOC=ON: use jemalloc
 * If USE_BUNDLED_JEMALLOC=OFF:
   * If jemalloc installed systemwide: use jemalloc
   * If jemalloc not installed: use libc malloc